conans/test/functional/scm/issues/test_ssh_user.py

```diff
@@ -0,0 +1,35 @@
+import textwrap
+import unittest
+
+import pytest
+
+from conans.model.ref import ConanFileReference
+from conans.test.utils.tools import TestClient, load
+
+
+@pytest.mark.tool_git
+class GitSSHTest(unittest.TestCase):
+
+    def test_ssh_username(self):
+        t = TestClient()
+        ref = ConanFileReference.loads("lib/version@issue/testing")
+        conanfile = textwrap.dedent("""
+            from conans import ConanFile
+            
+            class Conan(ConanFile):
+                scm = {
+                        "type": "git",
+                        "url": "ssh://github.com/conan-io/conan.git",
```

Comment:
```suggestion
                        "url": "ssh://git@github.com/conan-io/conan.git",
```

Reply:
with this test I wanted to reproduce the error in #8011. If I add the user to the URL, then the case is different

Reply:
Oh!, I see 👍